ssg/build_yaml.py

```diff
@@ -552,9 +552,12 @@ def validate_identifiers(self, yaml_file):
                 raise ValueError("Identifiers must not be empty: %s in file %s"
                                  % (ident_type, yaml_file))
             if ident_type[0:3] == 'cce':
-                if not is_cce_valid("CCE-" + ident_val):
-                    raise ValueError("CCE Identifiers must be valid: value %s for cce %s"
-                                     " in file %s" % (ident_val, ident_type, yaml_file))
+                if not is_cce_format_valid("CCE-" + ident_val):
+                    raise ValueError("CCE Identifier format must be valid: invalid format '%s' for CEE '%s'"
+                                     " in file '%s'" % (ident_val, ident_type, yaml_file))
+                if not is_cce_value_valid("CCE-" + ident_val):
+                    raise ValueError("CCE Identifier value must be valid: invalid value '%s' for CEE '%s'"
```

Comment:
If we want to, could say "CCE doesn't pass checksum validations" or something. Not sure if its worth changing though.

Reply:
@cipherboy updated the error message.